pkg/opsmanager/client.go

```diff
@@ -43,11 +43,11 @@
 package opsmanager
 
 import (
+	"errors"
```

Comment:
any reason we weren't using this in the first place?

Reply:
no idea :)